exporter/azuremonitorexporter/exporter_common.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package azuremonitorexporter
+
+// Contains code common to both trace and metrics exporters
+import (
+	"fmt"
+	"time"
+
+	"github.com/Microsoft/ApplicationInsights-Go/appinsights/contracts"
+	timestamp "github.com/golang/protobuf/ptypes/timestamp"
+)
+
```

Comment:
I'd split this into time.go, channels.go (or time_utils.go, channel_utils.go) etc instead of a single "common" file.